app/src/main/java/com/techcourse/controller/HomeController.java

```diff
@@ -0,0 +1,21 @@
+package com.techcourse.controller;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import nextstep.mvc.view.JspView;
+import nextstep.mvc.view.ModelAndView;
+import nextstep.web.annotation.Controller;
+import nextstep.web.annotation.RequestMapping;
+import nextstep.web.support.RequestMethod;
+
+@Controller
+public class HomeController {
+
+    private static final String INDEX_JSP = "index.jsp";
+
+    @RequestMapping(value = "/", method = RequestMethod.GET)
+    public ModelAndView execute(HttpServletRequest req, HttpServletResponse res) {
```

Comment:
피드백이라기보다는 제가 리팩토링하며 가독성이 더 좋아지는 부분에 대한 의견을 남길려고합니다!

```suggestion
    public String execute(HttpServletRequest req, HttpServletResponse res) {
```

현재 컨트롤러에서 일괄적으로 `ModelAndView`를 반환하기 때문에 Jsp를 사용하기 위해선 전부 `new JspView(...)`를 반복해서 사용하는 것 같아요!

이 부분을 `String`으로 반환하도록하고 AnnotationHandlerAdapter`부분에서 Controller에서 반환받은 값이 `String`이면 JspView로 반환하도록 하면 더 가독성이 좋아지더라구요!

스프링 코드를 참고하면서 리팩토링한 부분 중 가독성이 좋아졌던 부분이라 피드백 남깁니다~~

Reply:
감사합니다!
안 그래도 ModelAndView 및 JspView가 계속해서 생성되는 것 같아서 조금 마음이 불편했는데
리뷰 남겨주신 부분을 참고하여 반영하였습니다.
JsonView도 처리하려고하였지만, 일반적으로 Spring Boot에서 ResponseEntity로 반환을 지정해준다는 점을 참고하여 작업을 수행하지 않았습니다.
또한 테스트 코드도 조금 보강했는데, 인수 테스트는 톰캣과 같은 부분도 주입을 해주어야하는 것 같아서 조금 더 공부를 하도록 하겠습니다.
감사합니다! 마크